homeassistant/components/lyric/climate.py

```diff
@@ -342,78 +342,99 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
                     heatSetpoint=target_temp_low,
                     mode=mode,
                 )
-            except LYRIC_EXCEPTIONS as exception:
-                _LOGGER.error(exception)
+            except LYRIC_EXCEPTIONS as err:
+                raise HomeAssistantError("Failed to set heat/cool setpoints") from err
```

Comment:
This error is now actually user-facing. Let's make sure we add translations for it as well.